submissions/akinbobolaFaith/phpMySqlDatabase/customersList.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require_once("cookie.php");
+
+$query = new Query();
+
+if (isset($_COOKIE["customers"])) {
+    $data = json_decode($_COOKIE["customers"], true);
+    } else {
+    $query->allCustomers();
```

Comment:
I don't think this works. OR why do this `$query = new Query();`

Reply:
Let me tell you how to debug and be sure what you are doing is working.
- First, set the expire time to be very small, say 1 min
- Then use the `vardump()` function inside the `if else` statement. E,g
```
if (isset($_COOKIE["customers"])) {
      vardump(1); exit;
} else {
     vardump(2); exit;
}
```
- The above will help you know when the `if` or `else` block is executed. If you get `2`, it means the else block is executed
-  Now, try doing this `$query->allCustomers();` and see if it will work.

Reply:
Thanks for the debugging tip. I found it really helpful.